src/config.json

```diff
@@ -1258,6 +1258,70 @@
   ],
   "blacklist": [
     "xn--opensablog-vmb.com",
+    "airdrop.lacoste.cat",
+    "lacoste.cat",
+    "a156metamask.com",
+    "app-wallet-coinbase.googleystore.com",
+    "appuniswap.org",
+    "aptos-pancakeswap.finance",
+    "autholedgwallaps.org",
+    "authorise-coinbase.com",
+    "binance-exchange.com",
+    "binancewallet.org",
+    "coinbase-account49.dynamic-dns.net",
+    "coinbase-login-auth32.mefound.com",
+    "coinbase-secure-account.com",
+    "coinbase-sign-auth.dynamic-dns.net",
+    "coinbase.com---crypto.space",
+    "coinbaseverify.live",
+    "coinbasewallet.googleystore.com",
+    "dappsverificaion.work.gd",
+    "defiethereum.com",
+    "degods.trade",
+    "dgerguard.co",
+    "le.dgerguard.com",
+    "ethereum-connect.com",
+    "ethereum-defiaccess.com",
+    "ethereum-deficonnect.com",
+    "free-ethereum.tk",
+    "kyc-metamask-i0applicationoffceveriflcation10.neconme.com",
+    "login-kraken2022.com",
+    "login.coinbase.accounts.h7th.us",
```

Comment:
this and the 2 below could prob drop at least one subdomain to cast a wider net without false positives?